src/database/standalone-sqlite.ts

```
@@ -3776,13 +3819,13 @@ if (!isMainThread) {
           const ids = worker.getVerifiableDataIds();
           parentPort?.postMessage(ids);
           break;
-        case 'getRootTxIdFromCoreAndBundles':
-          const rootTxId = worker.getRootTxIdFromCoreAndBundles(args[0]);
-          parentPort?.postMessage(rootTxId);
+        case 'getRootTxFromCoreAndBundles':
+          const rootTx = worker.getRootTxFromCoreAndBundles(args[0]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)